pr-data.csv

```diff
@@ -2269,7 +2269,8 @@ https://github.com/apache/hadoop,a585a73c3e02ac62350c136643a5e7f6095a3dbb,hadoop
 https://github.com/apache/hbase,801fc05e9c082ab12c7c1207087d3667ab162d1a,hbase-server,org.apache.hadoop.hbase.io.TestMetricsIO.testMetrics,NIO,,,
 https://github.com/apache/hbase,801fc05e9c082ab12c7c1207087d3667ab162d1a,hbase-server,org.apache.hadoop.hbase.ipc.TestRpcMetrics.testSourceMethods,NIO,,,https://github.com/TestingResearchIllinois/idoft/issues/411
 https://github.com/apache/hbase,801fc05e9c082ab12c7c1207087d3667ab162d1a,hbase-server,org.apache.hadoop.hbase.master.cleaner.TestReplicationHFileCleaner.testIsFileDeletable,NIO,Accepted,https://github.com/apache/hbase/pull/2984,
-https://github.com/apache/hbase,801fc05e9c082ab12c7c1207087d3667ab162d1a,hbase-server,org.apache.hadoop.hbase.master.TestCatalogJanitor.testParentCleanedEvenIfDaughterGoneFirst,ID,,,
+https://github.com/apache/hbase,fe9a5e2eb91093e45d8f1d61f5ee45a1eca48414,hbase-server,org.apache.hadoop.hbase.master.janitor.TestCatalogJanitor.testParentCleanedEvenIfDaughterGoneFirst,ID,,,
+https://github.com/apache/hbase,801fc05e9c082ab12c7c1207087d3667ab162d1a,hbase-server,org.apache.hadoop.hbase.master.TestCatalogJanitor.testParentCleanedEvenIfDaughterGoneFirst,ID,MovedOrRenamed,,https://github.com/apache/hbase/pull/2373
```

Comment:
Please put in the `Notes` (last column) the commit SHA where this test (class) was moved to another package.